daprdocs/content/en/reference/components-reference/supported-middleware/middleware-oauth2.md

```diff
@@ -46,30 +50,20 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 
 | Field | Details | Example |
 |-------|---------|---------|
```

Comment:
This component is old format and needs to include a "Required" column. We can fill out the entries. This applies to several other middleware components, so check each one.